File: aoj/volume0/n0003/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();

        for (int i = 0; i < N; i++) {

            int a = sc.nextInt();
            int b = sc.nextInt();
            int c = sc.nextInt();

            if (a * a + b * b == c * c) {
                System.out.println("YES");
            } else if (a * a + c * c == b * b) {
                System.out.println("YES");
            } else if (b * b + c * c == a * a) {
                System.out.println("YES");
            } else {
                System.out.println("NO");
            }

        }

    }

}

File: aoj/volume0/n0003/Main.cpp
#include<iostream>

using namespace std;

int main() {

    int N;
    cin >> N;

    int a, b ,c;

    for (int i = 0; i < N; i++) {

        cin >> a >> b >> c;

        if (a * a + b * b == c * c) {
            cout << "YES" << endl;
        } else if (b * b + c * c == a * a) {
            cout << "YES" << endl;
        } else if (a * a + c * c == b * b) {
            cout << "YES" << endl;
        } else {
            cout << "NO" << endl;
        }

    }

    return 0;

}
